app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,145 @@
  *= require_tree .
```

Comment:
- [ ] While running your app I faced some issue that was blocking me from running your app and the problem was your last five migrations 😨 So I have fixed them by replacing the below code ⬇️ Kindly replace that too to make your app work as expected without any errors 👍

🔴 **Before**

```ruby
:user, null: false, foreign_key: true
```

🟢 **After**

```ruby
:author, null: false, foreign_key: { to_table: :users }
```